lib/transport/tls-context.c

```diff
@@ -498,6 +532,9 @@ tls_context_setup_context(TLSContext *self)
   if (_is_file_accessible(self, self->crl_dir) && !SSL_CTX_load_verify_locations(self->ssl_ctx, NULL, self->crl_dir))
     goto error;
 
+  if (!_set_client_ca_list(self))
```

Comment:
This is server only (`TM_SERVER `).

Reply:
According to https://www.openssl.org/docs/manmaster/man3/SSL_CTX_set_client_CA_list.html

>The "client CA list" functions below only have an effect when called on the server side.

This is why I have not checked which mode are we in, but I can add that explicitly as well. Would you like me to?

Reply:
I don't have a preference, I just remembered that we already have a few `TM_SERVER` checks in the code.

Reply:
Added the mode check just for good measure.